patches/chrome-android-java-src-org-chromium-chrome-browser-bookmarks-BookmarkUndoController.java.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkUndoController.java b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkUndoController.java
+index fad6de96cf7bd17a9eb7ab828cc97544cc94e422..a8384e71dc8935854b87ef4b6d85dac91a63656f 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkUndoController.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/bookmarks/BookmarkUndoController.java
+@@ -13,6 +13,7 @@ import org.chromium.chrome.browser.bookmarks.BookmarkModel.BookmarkDeleteObserve
+ import org.chromium.chrome.browser.snackbar.Snackbar;
+ import org.chromium.chrome.browser.snackbar.SnackbarManager;
+ 
++import java.util.List;
```

Comment:
just to make this patch a little smaller can you you do `private java.util.List<BookmarkItem> ...` here https://github.com/brave/brave-core/pull/4306/files#diff-eaf184844e7b3b93a3231427aaba23f6R17 ?

Reply:
done